src/core/server/config/raw_config_service.test.mocks.ts

```diff
@@ -19,6 +19,6 @@
 
 export const mockGetConfigFromFiles = jest.fn();
 
-jest.mock('./read_config', () => ({
+jest.mock('src/core/server/config/read_config', () => ({
   getConfigFromFiles: mockGetConfigFromFiles,
 }));
```

Comment:
(this commit will be deleted) This works even if `read_config` is imported using relative imports in test or sources `import XXX from '../foo/read_config'` (jest uses the resolved absolute path)